### PR TITLE
Pxc 8.0 3608

### PR DESCRIPTION
**Pull request by satya-bodapati**

### CODE REVIEW

File: storage/innobase/row/row0ins.cc [hunk 1]
```diff
@@ -1269,22 +1269,23 @@ static NO_INLINE MY_ATTRIBUTE((warn_unused_result)) dberr_t
...
 #ifdef WITH_WSREP
-  err = wsrep_append_foreign_key(thr_get_trx(thr), foreign,
-                                 cascade->pcur->m_old_rec, clust_index, false,
-                                 WSREP_SERVICE_KEY_EXCLUSIVE);
+  err =
+      wsrep_append_foreign_key(thr_get_trx(thr), foreign, clust_rec,
+                               clust_index, false, WSREP_SERVICE_KEY_EXCLUSIVE);
   if (err != DB_SUCCESS) {
     ib::warn() << "WSREP: foreign key append failed: " << err;
+    mtr_commit(mtr);
```
Comment (altmannmarcelo) [inline storage/innobase/row/row0ins.cc:1282]: @satya-bodapati what about if wsrep_append_foreign_key succeeds. When do we commit the mtr?

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: storage/innobase/row/row0ins.cc (hunk 1 above)
Comment (satya-bodapati) [reply storage/innobase/row/row0ins.cc:1282]: @altmannmarcelo there is mtr_commit() on both branches.

Comment (altmannmarcelo) [reply storage/innobase/row/row0ins.cc:1282]: ah, right, it will fall the else branch.

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Pull request merged**